### PR TITLE
metasploit: 6.0.26 -> 6.0.27

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Update to latest upstream release 6.0.27

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 110668` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>msf (metasploit)</li>
  </ul>
</details>

File: pkgs/tools/security/metasploit/gemset.nix [hunk 1]
```diff
@@ -1401,4 +1401,4 @@
     };
     version = "2.4.2";
   };
-}
+}
```
Comment (SuperSandro2000) [inline pkgs/tools/security/metasploit/gemset.nix:1404]: Please add that back.

Comment (fabaff) [reply pkgs/tools/security/metasploit/gemset.nix:1404]: Hmmm, that's wired...`update.sh` is generating this file and the script was not touched for months.

Comment (fabaff) [reply pkgs/tools/security/metasploit/gemset.nix:1404]: Ah, #110395 changed the file.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 110668` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>metasploit</li>
</ul>
</details>

File: pkgs/tools/security/metasploit/gemset.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/tools/security/metasploit/gemset.nix]: `update.sh` updated

File: pkgs/tools/security/metasploit/update.sh [hunk 2]
```diff
@@ -12,6 +12,7 @@ sed -i "s#refs/tags/.*#refs/tags/$latest\"#" Gemfile
 bundler install
 bundix
 sed -i '/[ ]*dependencies =/d' gemset.nix
+sed -i -e '$a\' gemset.nix
```
Comment (zowoq) [inline pkgs/tools/security/metasploit/update.sh:15]:
```suggestion
```
This isn't needed, it's already fixed in `bundix`.

Comment (fabaff) [reply pkgs/tools/security/metasploit/update.sh]: Indeed, thanks.

**Pull request merged**